src/schema/users.ts

```diff
@@ -245,6 +260,15 @@ export const resolvers: IResolvers<any, Context> = {
         };
       }
     },
+    userMostReadTags: async (
+      _,
+      { id }: { id: string },
+      ctx: Context,
+    ): Promise<GQLMostReadTag[]> => {
+      const user = await ctx.con.getRepository(User).findOneOrFail(id);
+
+      return getMostReadTags(ctx.con, user.id, { limit: 5 });
```

Comment:
Would we not want the frontend to specify the limit? 
Just wondering for other implementations.

Answer might also be: We'll deal with it when needed :)

Reply:
That's a very good point. I agree we can probably deal with it by the time comes and add a max limit since we usually want to get small amount of "most" data only